src/components/bulk-email-tool/bulk-email-form/BulkEmailForm.jsx

```diff
@@ -47,19 +64,26 @@ export default function BulkEmailForm(props) {
   };
 
   const onSubjectChange = (event) => setSubject(event.target.value);
+  const onDateTimeChange = (event) => {
+    const next = { [event.target.name]: event.target.value };
+    setDateTime((prev) => ({ ...prev, ...next }));
+  };
 
   const validateEmailForm = () => {
     const subjectValid = subject.length !== 0;
     const bodyValid = editorRef.current.getContent().length !== 0;
     const recipientsValid = selectedRecipients.length !== 0;
+    const { scheduleDate, scheduleTime } = dateTime;
+    const scheduleValid = isScheduled ? (scheduleDate && scheduleTime) !== undefined : true;
```

Comment:
I found this a little hard to read/grok at first. Just to make sure I _am_ reading it right... `isScheduled` is the condition and then we have a second conditional after the `?` --> (`(scheduleDate && scheduleTime) !== undefined`) that is determining the value if `isScheduled` is true? otherwise the conditional is false (if `isScheduled` is false?)?

I'd consider breaking it up into multiple lines if possible to make it easier to read, or trying to keep the conditional on the left side of the `?`.

Feel free to disagree and keep as-is. Just wanted to mention that I didn't find it entirely easy to read.

Reply:
yeah this is one of those things I built in pieces and didnt take the time to reread my draft. I can make this more stepped.